testgrid/config.yaml

```diff
@@ -976,6 +976,12 @@ test_groups:
 - name: ci-kubernetes-e2e-gci-gce-scalability-stable3
   gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gce-scalability-stable3
   num_failures_to_alert: 1
+- name: pull-kubernetes-e2e-gce-iscsi
+  gcs_prefix: kubernetes-jenkins/pr-logs/directory/pull-kubernetes-e2e-gce-iscsi
+  num_columns_recent: 20
+- name: pull-kubernetes-e2e-gce-iscsi-serial
```

Comment:
do we need ci jobs defined here too?

Reply:
I don't know, other ci- jobs don't have entries there (checked with `ci-kubernetes-e2e-gci-gce-slow`).

Reply:
I think the test_group_name specified in the dashboard (L4604, L4610) needs to have an entry here

Reply:
Ack, ci jobs added